Travel Scheduler/View Controllers/HomeCollectionViewController.m

```diff
@@ -209,22 +201,16 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
 
 - (NSIndexPath *)tableView:(UITableView *)tableView willSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    long cellNum = indexPath.row;
     MoreOptionViewController *moreOptionViewController = [[MoreOptionViewController alloc] init];
     moreOptionViewController.places = [[NSMutableArray alloc]init];
     moreOptionViewController.hub = self.hub;
-    if (cellNum == 0) {
-        moreOptionViewController.stringType = @"Parks";
-        moreOptionViewController.correctType = @"park";
-    } else if (cellNum == 1) {
-        moreOptionViewController.stringType = @"Museums";
-        moreOptionViewController.correctType = @"museum";
-    } else if (cellNum == 2) {
-        moreOptionViewController.stringType = @"Restaurants";
-        moreOptionViewController.correctType = @"restaurant";
-    } else if (cellNum == 3) {
-        moreOptionViewController.stringType = @"Hotels";
-        moreOptionViewController.correctType = @"lodging";
+    moreOptionViewController.correctType = self.arrayOfTypes[indexPath.row];
+    if ([moreOptionViewController.correctType isEqualToString:@"shopping_mall"]) {
+        moreOptionViewController.stringType = @"Mall";
+    } else {
+        NSString *firstCharacterInString = [[moreOptionViewController.correctType substringToIndex:1] capitalizedString];
+        NSString *capitalizedString = [moreOptionViewController.correctType stringByReplacingCharactersInRange:NSMakeRange(0,1) withString: firstCharacterInString];
+        moreOptionViewController.stringType = capitalizedString;
```

Comment:
This code looks really similar to the other part in the table view cell that also does the same thing of capitalizing strings. Idk if it's possible but you could consider either just passing the value from the table view cell to the moreoption view controller so that you don't have to do it twice or maybe making a helper method just for capitalization?